src/components/Landing/Landing.jsx

```diff
@@ -1,20 +1,30 @@
 import React from "react";
+import RENDER from "SHARED/renderUtils.jsx";
+import { s } from "./";
 
 export default function Landing(){
+
+	const bodyData = `When a king falls, another will rise. But who will it be?\nGet a closer look at `;
+
+	const body = RENDER.body(bodyData, "landing"); //comp name for key
+
 	return(
-		<header>
-			<h1>
+		<header className={ s.wrapper }>
+			<h1 className={ s.heading }>
 				Vikings
 			</h1>
-			<h2>
+			<h2 className={ s.subheading }>
 				Descend into Darkness
 			</h2>
-			<p>
-				When a king falls, another will rise. But who will it be?
-			</p>
-			<p>
-				Get a closer look at <span>Vikings season 5</span>.
-			</p>
+			<div className={ s.body }>
+				{ body }
+				<a 
```

Comment:
Another reason we'd want to take the markdown appraoch is that 'Vkings season 5' is part of the sentence in the paragraph above, so we'd want it to be a child, not a sibling